dagshub/data_engine/model/query_result.py

```diff
@@ -268,31 +267,49 @@ def get_blob_fields(
 
         # If no fields are specified, include all blob fields from self.datasource.fields
         if not fields:
-            fields = [field.name for field in self.datasource.fields if field.valueType == MetadataFieldType.BLOB]
-
-        for fld in fields:
-            logger.info(f"Downloading metadata for field {fld} with {num_proc} processes")
-            cache_location = self.datasource.default_dataset_location / ".metadata_blobs"
-
-            if cache_on_disk:
-                cache_location.mkdir(parents=True, exist_ok=True)
-
-            blob_urls = list(map(lambda dp: dp._extract_blob_url_and_path(fld), self.entries))
```

Comment:
Delete the now unused `_extract_blob_url_and_path`

Reply:
ty!!